### ISSUE TITLE
Solr backups occur hourly instead of nightly

### ISSUE THREAD

**Issue by andrew-morrison** [archivesspace/archivesspace#1618]
This, in config-defaults.rb, contains a discrepancy between what the comment says the setting does, and its actual setting value:
```
# By default, Solr backups will run at midnight.  See https://crontab.guru/ for
# information about the schedule syntax.
AppConfig[:solr_backup_schedule] = "0 * * * *"
```
"0 * * * *" means run at the zeroth minute of every hour.

## Expected Behavior
Run at midnight?

## Current Behavior
Runs hourly (confirmed in log files)

## Possible Solution
If the comment is correct, and it should run at midnight, the value should be changed to "0 0 * * *"

If the setting is correct, and hourly backups are desired, the comment should be changed for clarity.

## Steps to Reproduce (for bugs)
N/A

## Context
The backup uses Solr snapshot, so it should only take a few seconds, therefore running it hourly isn't a performance problem. But if people are timing their MySQL backups to also happen at midnight, the Solr backup will be newer.

## Your Environment
N/A

**Comment by alexduryee** [comment]
Agreed that the documentation and function should be in alignment.  Given that the comments specifically mention running backups at midnight, that seems to be what the cron should actually do.  I'll file a PR to adjust this - thanks for catching it!
